steward/src/cellars.rs

```diff
@@ -33,18 +33,22 @@ pub const ORACLE3: (U256, &str) = (
     U256([5, 0, 0, 0]),
     "26cde3f5db92ea91c84c838e664fe42dec1b6747",
 );
-pub const ALLOWED_PRICE_ORACLES: [(U256, &str); 2] = [ORACLE1, ORACLE2];
+pub const ORACLE4: (U256, &str) = (
+    U256([5, 0, 0, 0]),
+    "26cde3f5db92ea91c84c838e664fe42dec1b6747",
+);
+
+pub const ALLOWED_PRICE_ORACLES: [(U256, &str); 4] = [ORACLE1, ORACLE2, ORACLE3, ORACLE4];
 pub const ALLOWED_CACHE_PRICE_ROUTER: [&str; 1] = [CELLAR_RYETH];
 
 // permissions
 
 pub const ALLOWED_V2_0_SETUP_ADAPTORS: [(&str, &str); 1] = [(CELLAR_RYUSD, ADAPTOR_CELLAR_V2)];
-pub const ALLOWED_V2_2_CATALOGUE_ADAPTORS: [(&str, &str); 3] = [
-    (CELLAR_RYBTC, ADAPTOR_LEGACY_CELLAR_V1),
-    (CELLAR_RYBTC, ADAPTOR_COLLATERAL_F_TOKEN_V1),
-    (CELLAR_RYBTC, ADAPTOR_DEBT_F_TOKEN_V1),
+pub const ALLOWED_V2_2_CATALOGUE_ADAPTORS: [(&str, &str); 0] = [];
```

Comment:
Have we confirmed all the 2.2 adaptor add calls were made?

Reply:
No, looking for feedback from them in this PR